src/main/java/com/bandwidth/sdk/messaging/BandwidthMessagingException.java

```diff
@@ -0,0 +1,49 @@
+package com.bandwidth.sdk;
+
+
+public class BandwidthMessagingException extends RuntimeException {
```

Comment:
Maybe just `MessagingException`? Bandwidth seems a bit redundant to me, but I don't have strong feelings

Reply:
Makes sense. Changed